src/fluent-plugin-mdsd/test/plugin/test_out_mdsd.rb

```diff
@@ -48,6 +49,7 @@ def test_configure()
         assert_equal("/tmp/mytestsocket", d.instance.djsonsocket, "djsonsocket")
         assert_equal(1, d.instance.acktimeoutms, "acktimeoutms")
         assert_equal([ "^mdsd.syslog" ], d.instance.mdsd_tag_regex_patterns, "mdsd_tag_regex_patterns")
```

Comment:
>^mdsd.syslog [](start = 24, length = 12)

(Unrelated) It's weird how this test passed, when the string here is not an exact match with the CONFIG1/2 above (missing '\\')...